apps/nextjs-website/src/components/atoms/Stats/Stats.tsx

```diff
@@ -32,40 +32,39 @@ const Stats = ({ items, maxWidth = 150, useDarkTheme = false }: StatsProps) => {
           justifyContent={'space-between'}
           alignItems={'center'}
           gap={4}
-          paddingTop={{ xs: '2.5rem', sm: 12 }}
-          paddingBottom={{ xs: '2.5rem', sm: 13 }}
+          paddingTop={{ xs: '2.5rem', sm: '75px' }}
+          paddingBottom={{ xs: '2.5rem', sm: '104px' }}
           width={'100%'}
         >
           {items.map(({ title, subtitle, description }, index) => (
             <Stack
               key={index}
               flex={1}
-              gap={1}
               flexDirection={'column'}
               textAlign={'center'}
               justifyContent={'start'}
               maxWidth={maxWidth}
             >
               <Typography
-                marginBottom={3}
+                marginBottom={'.7rem'}
+                fontSize={50}
                 fontWeight={700}
                 color={titleColor}
-                variant='h3'
               >
                 {title}
               </Typography>
               {subtitle && (
```

Comment:
@marcobottaro i notices that on strapi, there is no subtitle for the stats in the solution model, so this part of code is never used